src/main/java/com/krystian/Fifth.java

```diff
@@ -0,0 +1,190 @@
+package com.krystian;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+import java.util.regex.Pattern;
+
+public class Fifth {
+    private List<String> allLinesFromText = new ArrayList<>();
+    private List<List<String>> boxes = new ArrayList<>();
+
+    public List<List<String>> getBoxes() {
+        return boxes;
+    }
+
+    public void fillOutAllLinesFromTextArrayWithLines() throws IOException {
```

Comment:
nie uczylem o wyjatkach ? ze nie uzywamy throws IOException ani innych  takich rzeczy tyle razy? 
Bo u Ciebie to w kodzie kazda metoda prawie to robi. Takie cos nie moze miec miejsca